pyanaconda/network.py

```diff
@@ -116,8 +126,11 @@ def sanityCheckHostname(hostname):
 
     return (True, "")
 
-# Return a list of IP addresses for all active devices.
 def getIPs():
+    """ Return a list of IP addresses for all active devices.
+
+        :return: list of IP addresses
```

Comment:
I think this only duplicates the description. Either drop it and make the docstring one-line or replace it with ``:rtype:`` which would be more interesting here, I'd say.